integration/run_test

```diff
@@ -69,6 +70,21 @@ then
     )
 fi
 
+test_commandline() {
+    local env_vars
+    if [[ -n "$VAULT_UTIL_SECRETS" ]]; then
+        echo "    # When on the Chef VPN"
+        echo "    CHEF_USERNAME=yourusername scripts/get_secrets.sh"
+        echo "    export VAULT_ADDR=https://vault.chef.co:8200"
+        jq -r 'to_entries | map("    export " + .key + "=$(vault kv get -field=" + .value.field + " " + .value.path + ")")[]' <<<"$VAULT_UTIL_SECRETS"
```

Comment:
This is kinda a bummer but I guess a few tests require a number of secrets.  If it were just the A2_LICENSE we could probably simplify this generation. But oh well 🤷‍♂